### ISSUE TITLE
macOS binaries for v1.4.2 fail with `dyld[3912]: Library not loaded: @rpath/libtbb.12.dylib`

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/nextclade#595]
Reported from a user on intel mac OS 10.14 running the latest pre-built Nextalign binaries (v 1.4.2)

```
dyld: Library not loaded: @rpath/libtbb.12.dylib
 Referenced from: /Users/jwel033/Desktop/seqs/nextalign-MacOS-x86_64
 Reason: image not found
Abort trap: 6
```

Have you seen this before @ivan-aksamentov / any tips on how to debug?

**Comment by corneliusroemer** [comment]
Haven't seen but two tips:
1. Update macOS. `10.14 = Mojave` is no longer supported
2. Try `brew upgrade node` (if you haven't got Homebrew installed, get it), source: https://stackoverflow.com/questions/17703510/dyld-library-not-loaded-reason-image-not-found was pretty easy to find just googling the first line of the error message

**Comment by corneliusroemer** [comment]
I can reproduce 😕 

The new macOS Intel binaries do not seem to work. Both nextalign and nextclade fail with the following error:
```
❯ assets/nextclade-MacOS-x86_64 -v
dyld[3912]: Library not loaded: @rpath/libtbb.12.dylib
  Referenced from: /Users/cr/code/nextclade/assets/nextclade-MacOS-x86_64
  Reason: tried: '/usr/local/lib/libtbb.12.dylib' (no such file), '/usr/lib/libtbb.12.dylib' (no such file)
fish: Job 1, 'assets/nextclade-MacOS-x86_64 -v' terminated by signal SIGABRT (Abort)
```

The release assets for 1.4.1 work without problem.

My system details: macOS 12.0.1 Monterey

**Comment by corneliusroemer** [comment]
I can fix it by installing `tbb` through `brew install tbb` but should we add a dependency to nextclade that requires manual installing through `brew`? Probably not a good idea.

```
❯ assets/nextclade-MacOS-x86_64 -v
dyld[10937]: Library not loaded: @rpath/libtbb.12.dylib
  Referenced from: /Users/cr/code/nextclade/assets/nextclade-MacOS-x86_64
  Reason: tried: '/usr/local/lib/libtbb.12.dylib' (no such file), '/usr/lib/libtbb.12.dylib' (no such file)
fish: Job 1, 'assets/nextclade-MacOS-x86_64 -v' terminated by signal SIGABRT (Abort)

nextclade on  docs/citation-suggestion [$] via △ v3.21.4 via  via 🐍 v3.9.7 
❯ brew install tbb
==> Downloading https://ghcr.io/v2/homebrew/core/tbb/manifests/2021.4.0
Already downloaded: /Users/cr/Library/Caches/Homebrew/downloads/6dd4f8f6334fd9e4e3035563bbd5a7c7c0831828e072477aa852869b4957f22d--tbb-2021.4.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/tbb/blobs/sha256:dcbc8429653c9953eeb40e73f228ebbdfca7e86bbcc13dd
Already downloaded: /Users/cr/Library/Caches/Homebrew/downloads/beee6091c6b5e83c4f8b5629e7c5c518963c38671fdc83dff0d58d74a908837c--tbb--2021.4.0.monterey.bottle.tar.gz
==> Pouring tbb--2021.4.0.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/tbb/2021.4.0: 185 files, 2MB

nextclade on  docs/citation-suggestion [$] via △ v3.21.4 via  via 🐍 v3.9.7 took 4s 
❯ assets/nextclade-MacOS-x86_64 -v
1.4.2
```

**Comment by ivan-aksamentov** [comment]
Whoops! My bad. I hope this resolves it: https://github.com/nextstrain/nextclade/pull/596

I've set up staging builds for this version and they should hopefully appear here in a few:
https://app.circleci.com/pipelines/github/nextstrain/nextclade/1688/workflows/cc17dfd8-12eb-4037-b9d1-bb19786ff598/jobs/901/artifacts

https://app.circleci.com/pipelines/github/nextstrain/nextclade/1688/workflows/cc17dfd8-12eb-4037-b9d1-bb19786ff598/jobs/900/artifacts

If you guys have a second please double check the arm and x86 versions, and x86 under Rosetta 2, where possible.

The cleaner the environment the better, i.e. no tbb or node or anything at all should be required. IT should run as is on a freshly purchased mac.

If all goes well, I'll release.

Thanks!

**Comment by ivan-aksamentov** [comment]
I merged https://github.com/nextstrain/nextclade/pull/596 and released as 1.4.4. As Cornelius said, it cannot do any worse 😆  

The binaries should be on [GitHub releases](https://github.com/nextstrain/nextclade/releases) in a few minutes. 

Still wondering how it does on arm.
